.nvmrc

```
@@ -1 +1 @@
-v16
+v18.18.0
```

Better to specify the latest LTS version available
```suggestion
v18.20.5
```